cmake/HalidePythonExtensionHelpers.cmake

```diff
@@ -28,7 +28,7 @@ set(_STUB_DIR "${Halide_SOURCE_DIR}/python_bindings/stub")
 # suffixed with "_stub" for Stub extensions. (TODO: this is unsatisfyingly hackish; better suggestions
 # would be welcome.)
 
-function(target_export_single_symbol TARGET SYMBOL)
+function(_target_export_single_symbol TARGET SYMBOL)
     configure_file("${_STUB_DIR}/ext.ldscript.apple.in" "${CMAKE_CURRENT_BINARY_DIR}/${TARGET}.ldscript.apple")
     configure_file("${_STUB_DIR}/ext.ldscript.linux.in" "${CMAKE_CURRENT_BINARY_DIR}/${TARGET}.ldscript")
     target_export_script(
```

Comment:
This function is not built into CMake, but is rather one of our helpers, which is not already installed. That will need to be added to the package as well.

Reply:
Done